src/transformStateWithClones.js

```diff
@@ -64,8 +64,35 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const newState = Object.assign({}, state);
```

Comment:
```suggestion
  const newState = { ...state };
```
You can simply use spread-operator here